### PR TITLE
Add range to find command

### PR DESCRIPTION
**Pull request by TPH777**
Add optional `to` after the default format to search within a range

Change readCommad such that all whitespaces will be treated as one for splits to be done more reliably through whitespaces only

### CODE REVIEW

**Review by wx-03 — COMMENTED** [review]
Maybe can throw an error when first value > second value ?

**Comment by TPH777** [comment]
> Maybe can throw an error when first value > second value ?

Ok done

File: src/main/java/seedu/commands/FindCommand.java [hunk 1]
```diff
@@ -91,22 +96,56 @@ private <T extends EntryType> ArrayList<T> findList(String[] arguments, ArrayLis
...
+    private <T extends EntryType> ArrayList<T> getMatchingDescription(String findValue, ArrayList<T> list) {
+        return list.stream()
+                .filter(entry -> entry.getDescription().contains(findValue))
+                .collect(Collectors.toCollection(ArrayList::new));
+    }
+
+    private <T extends EntryType> ArrayList<T> getMatchingDate(String findValue, ArrayList<T> list) {
+        LocalDate lower;
+        LocalDate upper;
+        if (findValue.contains("to")) {
```
[the remark this answers is not in the document]
Comment (TPH777) [reply src/main/java/seedu/commands/FindCommand.java]: Settled

**Comment by wx-03** [comment]
lgtm!

**Pull request merged**